### ISSUE TITLE
Copy selected text on the dictionary page to “Translation” automatically

### ISSUE THREAD

**Issue by GrimPixel** [jzohrab/lute-v3#211]
**Is your feature request related to a problem? Please describe.**

The current way requires some key-pressing to add the translation.

**Describe the solution you'd like**

Copy selected text on the dictionary page to “Translation” automatically

**Additional context**

[VocabSieve](https://github.com/FreeLanguageTools/vocabsieve) supports the feature that add the translation by clicking on the word directly for some dictionary websites.

**Comment by jzohrab** [comment]
Right, this would be nice.  I don't know how to do it though, perhaps someone with more js skillz can weigh in.

**Comment by jzohrab** [comment]
I believe that VocabSieve does this by parsing dictionary results and presenting its own screen, adding some actions and handlers.  That's too much work, with all of the dictionaries out there.

There appear to be security restrictions on accessing the clipboard via js in iframe embedded dictionaries.  Google "javascript clipboard iframe different origin" gave a few hits.  Maybe someone can take this info and look into it, I don't have time for it myself.  Cheers!
